_data/people.yaml

```diff
@@ -6453,6 +6459,12 @@ mgawe-cavin:
   continent: Africa
   longitude: 36.8288420082562
   latitude: -1.3026148499999999
+mgomezn:
+  first-name: MARIANA PATRICIA
+  last-name: GOMEZ NICOLAS
```

Comment:
Can you put the names as capitalized (so only 1st letter as upper case)?
Note for latter: we need to modify the script so it does it

Reply:
For sure! 💯

Afterwards, I can also modify the scripts and PR on a different branch.